python/export_util.py

```diff
@@ -235,3 +311,287 @@ def csv_query(
         data = csv_to_stream(data_list)
 
     return mgp.Record(file_path=file_path, data=data)
+
+
+def write_header(output: io.StringIO):
+    output.write('<?xml version="1.0" encoding="UTF-8"?>\n')
+    output.write(
+        '<graphml xmlns="http://graphml.graphdrawing.org/xmlns" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://graphml.graphdrawing.org/xmlns http://graphml.graphdrawing.org/xmlns/1.0/graphml.xsd">\n'  # noqa: E501
+    )
+
+
+def translate_types(variable: Any):
+    if isinstance(variable, str):
+        return "string"
+    if isinstance(variable, bool):
+        return "boolean"
+    if isinstance(variable, float):
+        return "float"
+    if isinstance(variable, int):
+        return "int"
+    raise Exception(
+        "Property values can only be primitive types or arrays of primitive types."  # noqa: E501
+    )
+
+
+def check_if_elements_same_type(variable: Any):
+    for element in variable:
+        if not isinstance(element, type(variable[0])):
```

Comment:
Extract `type(variable[0])` ahead of the for loop to avoid calling it more than once.